code/components/citizen-scripting-core/src/ScriptHost.cpp

```diff
@@ -340,7 +340,7 @@ result_t SAFE_BUFFERS TestScriptHost::InvokeNative(fxNativeContext & context)
 	}
 	else
 	{
-		trace("WARNING: NON-EXISTENT NATIVE %016llx\n", context.nativeIdentifier);

```

Comment:
Another thing to note is this gets called if *any* native is called incorrectly, saying they're 'not supported' is, imo, less useful than saying the native you tried to call didn't exist